code/modules/mob/living/carbon/carbon_defines.dm

```diff
@@ -31,6 +31,8 @@
 	var/cpr_time = 1.0
 	nutrition = 400.0//Carbon
 
+	var/is_watching = TRUE  //used for remote viewing of multiz structures
+
```

Comment:
is_watching should be a signal but seeing as this is standing behavior Ill let @KathyRyals decide, though this might need a minor refactor